procursus-deploy-linux-macos.sh

```diff
@@ -26,16 +35,20 @@ if [[ "${ARM}" = yes ]]; then
 		exit 1
 	fi
 else
-	if which iproxy >> /dev/null; then
-		iproxy 4444 44 >> /dev/null 2>/dev/null &
-	else
-		echo "Error: iproxy not found"
-		exit 1
+	cd $TMP
+	curl -L -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/usbmux.pyc -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/usbmux.py -O https://github.com/M1staAwesome/Odyssey-bootstrap/raw/master/tcprelay.py
```

Comment:
Make sure these change to the upstream url before this gets merged

Reply:
completely forgot about that, thanks